tmp.txt

```diff
@@ -1 +1,4 @@
 Placeholder to create repo from web, rather than command line instructions.
+
+flib
+foo
```

Comment:
What about flam and flub?
<img width="868" alt="Screen Shot 2021-02-18 at 8 29 10 PM" src="https://user-images.githubusercontent.com/847448/108446570-59768880-722c-11eb-8e34-e5766876e06b.png">

Reply:
done in 9145f3b